src/Folder.php

```diff
@@ -90,6 +90,13 @@ public function onBeforeDelete()
         parent::onBeforeDelete();
     }
 
+    public function onBeforeWrite()
+    {
+        parent::onBeforeWrite();
+
+        $this->Title = $this->getField('Name');
```

Comment:
So you can still manually change the Title field because this is in a beforeWrite right?